tests/yast2_cmd/yast_nfs_server.pm

```diff
@@ -40,7 +40,7 @@ sub check_bsc1142979 {
     my $grep_cmd = script_run("grep -i $bsc_dir /etc/exports");
     if ($grep_cmd != 0) {
         my $ret_val = script_run("yast nfs-server delete mountpoint=$bsc_dir");
-        if ($ret_val == 0) {
+        if ($ret_val == 0 || $ret_val == 16) {
```

Comment:
Why it is bug when `$ret_val == 0` ?

Reply:
In the sles=>12sp3 system, the yast command returns 0 when removing a non-existent mount point, and 16 in sles12sp2, which is not the expected return value. So bsc#1142979 checks for these return values.

Reply:
ok